transcom/mymove
# B-18154 fix submitting ppm with zero allowance

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A871783&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

Service Counselors were able to submit a PPM with pro gear allowances less than the amounts entered in the move.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the office app
2. Login as a service counselor
3. Open a move with a PPM that has Pro Gear.
4. Edit the allowances to be LESS than the amounts entered into the move.
5. The submit button should be disabled and an error message should appear under the Move Details label.

### Frontend
